app/controllers/concerns/with_custom_assets.rb

```diff
@@ -0,0 +1,9 @@
+module WithCustomAssets
+  def theme_stylesheet_url
+    Organization.current.theme_stylesheet_url
```

Comment:
You don't need to get the current organization, Organization class includes ConfigurableGlobal module